### PR TITLE
Allows admins to edits expired and removed bans

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request
Untested and webeditpilled :sunglasses: 

ported: https://github.com/tgstation/tgstation/pull/59431

## Changelog
:cl:
admin: Admins can edits expired and removed bans
/:cl:

### CODE REVIEW

**Comment by Archanial** [comment]
oh my god why is copypasting so hard I swear I didn't drink that much

**Comment by Archanial** [comment]
please pass

**Comment by AffectedArc07** [comment]
But why? This seems like a bad idea

**Comment by Archanial** [comment]
Because sometimes bans get appealed and we want to change their message.

**Review by Crossedfall — DISMISSED** [review]
👏

File: code/modules/admin/sql_ban_system.dm [hunk 1]
```diff
@@ -741,8 +741,7 @@
 			if(unban_datetime)
 				output += "<br>Unbanned by <b>[unban_key]</b> on <b>[unban_datetime]</b> during round <b>#[unban_round_id]</b>."
 			output += "</div><div class='container'><div class='reason'>[reason]</div><div class='edit'>"
-			if(!expired && !unban_datetime)
-				output += "<a href='?_src_=holder;[HrefToken()];editbanid=[ban_id];editbankey=[player_key];editbanip=[player_ip];editbancid=[player_cid];editbanrole=[role];editbanduration=[duration];editbanadmins=[applies_to_admins];editbanreason=[rustg_url_encode(reason)];editbanpage=[page];editbanadminkey=[admin_key]'>Edit</a><br>[unban_href]"
+			output += "<a href='?_src_=holder;[HrefToken()];editbanid=[ban_id];editbankey=[player_key];editbanip=[player_ip];editbancid=[player_cid];editbanrole=[role];editbanduration=[duration];editbanadmins=[applies_to_admins];editbanreason=[url_encode(reason)];editbanpage=[page];editbanadminkey=[admin_key]'>Edit</a><br>[unban_href]"
```
Comment (Crossedfall) [inline code/modules/admin/sql_ban_system.dm]: You still didn't copy paste this correctly. You're not using our output

**Review by Crossedfall — CHANGES_REQUESTED** [review]

File: code/modules/admin/sql_ban_system.dm (hunk 1 above)
Comment (Archanial) [inline code/modules/admin/sql_ban_system.dm]:
```suggestion
			output += "<a href='?_src_=holder;[HrefToken()];editbanid=[ban_id];editbankey=[player_key];editbanip=[player_ip];editbancid=[player_cid];editbanrole=[role];editbanduration=[duration];editbanadmins=[applies_to_admins];editbanreason=[rustg_url_encode(reason)];editbanpage=[page];editbanadminkey=[admin_key]'>Edit</a><br>[unban_href]"
```

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**